ethpm/package.py

```diff
@@ -11,27 +12,69 @@
 
 
 class Package(object):
-    def __init__(self, lockfile):
+    web3 = None
+
+    def __init__(self, package_identifier, web3=None):
         """
         A lockfile can be:
         - filesystem path
         - parsed lockfile JSON
         - lockfile URI
         """
-        if not os.path.exists(lockfile):
+        self.web3 = web3
+
+        if not os.path.exists(package_identifier):
             raise ValidationError
 
-        self.package_identifier = lockfile
+        self.package_identifier = package_identifier
 
-        schema_data = json.load(open('./ethpm/lockfileSpecification.json'))
-        package_data = json.load(open(lockfile))
+        schema_data = self._load_json('./ethpm/assets/release-lockfile.schema.v1.json')
+        package_data = self._load_json(package_identifier)
 
         try:
             validate(package_data, schema_data)
         except jsonValidationError:
             raise ValidationError
         self.package_data = package_data
 
+    def _load_json(self, path):
```

Comment:
This falls into the category of *utility* which means I'd relegate it to something under `ethpm.utils`.  My suggestion would be to split this out into two independent functions.

* one that is dedicated to loading schemas and knows *where* to find them so that you don't have to provide absolute paths (HINT: look in populus for the `ASSETS_DIR` as I'd suggest modeling it after that.)
* one that is dedicated to loading packages from disk as JSON files (which probably does exactly what this function does).

A good guideline is to check whether `self` is used in a method.  If it isn't the thing probably deserves to get split out into it's own function.  (same for `cls` with `classmethods`).